coursemapper-kg/recommendation/app/views/course_materials.py

```
@@ -85,89 +67,19 @@
-        user_id=user_id,
-        slide_id=slide_id,
-        material_id=material_id,
-        recommendation_type=recommendation_type,
-    )
-
-    return resp
+def get_resources(job):
+    data = job["data"]
+    resource_recommender_service = ResourceRecommenderService()
+    result = resource_recommender_service._get_resources(data_default=data["default"], data_rec_params=data["rec_params"])
+    return result
```

Ruff (0.12.2)

Unnecessary assignment to `result` before `return` statement

Remove unnecessary assignment

(RET504)